modules/nixos/emergency-access.nix

```diff
@@ -0,0 +1,4 @@
+{
+  # allow to access emergency shell with a password
+  boot.initrd.systemd.emergencyAccess = "$6$he2fblfl/H7I.kvz$WbSCMXu8ztmqfj5jG4czqvu/rkMHxufxqHgy1urzXFSN.jZB4QiW5lOjR08vk8pZTyim3TT1wFkMaNE9zZ3sc1";
```

Comment:
@Mic92 Should we have this enabled by default?

Reply:
Potentially. Back then, I was using `disko-install` to enable it.

Reply:
We can also enable ssh in the initrd as well so that we don't need to order a kvm switch every time.